.github/workflows/dev.yml

```diff
@@ -40,17 +40,16 @@ jobs:
     name: Use prettier to check formatting of documents
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-node@v4
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683  # v4.2.2
```

Comment:
Is there a reason we have this specific hash? If there is, can we have a comment to note that? Also, looks like there is version 5 now